.travis.yml

```diff
@@ -31,6 +31,7 @@ matrix:
 
 before_install:
   - composer self-update
+  - composer clear-cache
```

Comment:
Why was this change made? Seems completely unnecessary.

Reply:
The head is here: 
Change:
```
        return new RollbarHandler(Rollbar::logger(), LogLevel::ERROR);
To:
        return new RollbarHandler(Rollbar::logger(), $this->minimumLevel);
```
Because there are errors above the error equation, but in config lib have params minimumLevel log.